engine-nextgen/src/main/kotlin/graphql/nadel/validation/NadelUnionValidation.kt

```diff
@@ -0,0 +1,45 @@
+package graphql.nadel.validation
+
+internal class NadelUnionValidation(
+    private val typeValidation: NadelTypeValidation,
+) {
+    fun validate(
+        schemaElement: NadelServiceSchemaElement,
+    ): List<NadelSchemaValidationError> {
+        return emptyList()
+
+        // return if (schemaElement.overall is GraphQLUnionType && schemaElement.underlying is GraphQLUnionType) {
```

Comment:
Couldn't quite wrap my head around the logic here. Will revisit later.

Reply:
We have this same code in the gateway, right? Are we leaving a gap in our validation by commenting this out?

Reply:
I don't see any validation on it, so don't think it's a big deal.